src/operator/c_lapack_api.h

```diff
@@ -325,41 +325,30 @@ inline void flip(int m, int n, DType *b, int ldb, DType *a, int lda) {
 #else
 
   // use pragma message instead of warning
+  /*
```

Comment:
Why commenting out? Maybe it should go to `c_lapack_api.cc`?

Reply:
Good point